com.unity.multiplayer.mlapi/Tests/Runtime/TickSystemTests.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.TestTools;
+using NUnit.Framework;
+
+namespace MLAPI.RuntimeTests
+{
+    public class TickSystemTests: IDisposable
+    {
+        private NetworkTickSystem m_TickSystem = null;
+        private float m_TestDuration = 5.0f;
+        private float m_SleepInterval = 0.001f;
+        private float m_TickInterval = 0.010f;
+
+        public void Dispose()
```

Comment:
minor: you might want to consider using `[TearDown] void Teardown() { ... }` instead of `Dispose()` but both would work fine.

Reply:
https://docs.nunit.org/articles/nunit/technical-notes/usage/SetUp-and-TearDown.html